book-1-orientation/chapters/HTTPCLIENT.md

```diff
@@ -0,0 +1,153 @@
+# HttpClient
+
+You've already experienced making http requests from your browser via `fetch`. While building web applications, we often want to make those requests from our server side code. .NET provides us with the `HttpClient` class that allows us to do that.
+
+We'll first need to install an additional package to make this work. Run the following command in your project (NOTE: This is required only if you are in a Console application. If you are in an ASP.NET project, this is not necessary):
+
+```sh
+dotnet add package Microsoft.AspNet.WebApi.Client
+```
+
+
+## Why Make Requests on the Server?
+In a full stack web application, we know we can make requests to 3rd party APIs from the our client side javascript code. So why would we instead choose to make those calls from our server side code? Some of the benefits of making Http requests server side as opposed to client side are:
+
+- Avoid CORS issues
+- Secure sensitive credentials like keys and passwords by not exposing them in your client side code
+- Users can already have external data available as soon as your page loads
+
```

Comment:
These are good reasons. I’d also add something about caching data across multiple users/sessions.
Maybe...add something about calling an api that’s internal to your network too.